lib/cartopy/tests/test_geodesic.py

```diff
@@ -0,0 +1,54 @@
+# (C) British Crown Copyright 2015, Met Office
+#
+# This file is part of cartopy.
+#
+# cartopy is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cartopy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with cartopy.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+
+import unittest
+
+import numpy as np
+from numpy.testing import assert_almost_equal, assert_array_almost_equal
+from nose.tools import assert_equal
+
+from cartopy import geodesic
+
+
+class TestGeodesic(unittest.TestCase):
+
+    def test_dir(self):
+        geod = geodesic.Geodesic()
+        geod_dir = geod.direct([40, 50], 38, 2000)
+        assert_almost_equal(geod_dir, np.array([[40.01717933, 50.01416786,
```

Comment:
> I will add that to the doc, but keep it as forward I think, as that is how PROJ4 does it.

OK no problem. At the moment it just says azimuth, and since I've only veer come across this using backwards azimuths (because it tells you which direction to go back to get where you started) so I assumed this is what you meant, so the documentation needs to be quite clear on this.